migrations/2024-10-30-155932_increase_post_url_max_length/up.sql

```diff
@@ -3,3 +3,7 @@
 ALTER TABLE post
     ALTER COLUMN url TYPE varchar(2000);
 
+DROP INDEX idx_post_url;
+
+CREATE INDEX idx_post_url ON post USING HASH (url);
```

Comment:
It'd also be good to run 

`vacuum full analyze post;`

based on @phiresky 's recomendations